test/feed.test.js

```diff
@@ -54,5 +62,21 @@ describe('Fost data class tests', () => {
       const feed = await Feed.byUrl('http://invalid.url.com');
       expect(feed).toBe(null);
     });
+
+    test('Setting etag on a feed should persist', async () => {
+      const feed = new Feed('author', 'http://url.com/with/etag');
+      feed.etag = 'etag';
+      await feed.save();
+      const persisted = await Feed.byId(feed.id);
+      expect(persisted.etag).toEqual('etag');
+    });
+
+    test('Setting lastModified on a feed should persist', async () => {
+      const feed = new Feed('author', 'http://url.com/with/lastModified');
+      feed.lastModified = 'lastModified';
+      await feed.save();
+      const persisted = await Feed.byId(feed.id);
+      expect(persisted.lastModified).toEqual('lastModified');
```

Comment:
should we have a tag here to test a feed without an etag and expect null? We saw a case where etag didn't show up in the header when we used my feed right?

Reply:
Good idea.  I added two more expects to deal with this in a new commit.